app/controllers/export_controller.rb

```
@@ -0,0 +1,151 @@
+  }.freeze
+
+  before_action :export_params
+
+  def resolve_export_class(name)
+    SUPPORTED_EXPORT_CLASSES[name.to_s]
+  end
+
+  def index
+    klass = resolve_export_class(params[:class])
+    raise ArgumentError, "Unsupported export class: #{params[:class]}" if klass.nil?
+    
```

RuboCop (1.86.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)